src/ray/core_worker/store_provider/store_provider.h

```diff
@@ -0,0 +1,65 @@
+#ifndef RAY_CORE_WORKER_STORE_PROVIDER_H
+#define RAY_CORE_WORKER_STORE_PROVIDER_H
+
+#include "ray/common/buffer.h"
+#include "ray/common/id.h"
+#include "ray/common/status.h"
+#include "ray/core_worker/common.h"
+
+namespace ray {
+
+/// Provider interface for store access. Store provider should inherit from this class and
+/// provide implementions for the methods. The actual store provider may use a plasma
+/// store
+/// or local memory store in worker process, or possibly other types of storage.
+
```

Comment:
remove this blank line

Reply:
yep. This is auto-added by clang-format:)